### PR TITLE
Add router and a dummy hospital search

### PR DESCRIPTION
**Pull request by bl-nero**

### CODE REVIEW

File: src/components/HospitalSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState, FormEvent } from 'react';
```
Comment (kaashmonee) [inline src/components/HospitalSearch.tsx:1]: Could you clarify what the code is doing here by adding a bit of documentation?

Comment (bl-nero) [reply src/components/HospitalSearch.tsx:1]: Sure, will do.

Comment (bl-nero) [reply src/components/HospitalSearch.tsx:1]: @kaashmonee How about now?

Comment (kaashmonee) [reply src/components/HospitalSearch.tsx:1]: Yep, this looks great! Thanks!

**Pull request merged**